### DISCUSSION TITLE
能否/如何将实时网速的B显示为K？

### DISCUSSION THREAD

**Discussion by gundagai** [zdz/ServerStatus-Rust#127] [Ideas]
由于类似990.0B这样的长度比较长，加上刷新速度较快，时不时会改动行高，在节点较多，窗口不宽的时候非常容易导致看串行。如果转换成0.1K这样就会好很多。

**Comment by zdz** [comment]
哪个手机型号？这个特意调过，高度不会动才对

**Comment by gundagai** [comment]
谢谢回复! 型号是iPad Pro 11寸, Chrome浏览器. iPhone上貌似没有问题.

**Comment by zdz** [comment]
后面调调

**Comment by zdz** [comment]
试试 [v1.7.0](https://github.com/zdz/ServerStatus-Rust/releases/tag/v1.7.0)
